src/Stryker.Core/Stryker.Core/Mutants/NodeOrchestrators/NodeSpecificOrchestrator.cs

```diff
@@ -89,35 +94,94 @@ protected NodeSpecificOrchestrator(CsharpMutantOrchestrator mutantOrchestrator)
         /// <remarks>Override this method if you want to control how the node's children are mutated. simply return <see cref="node"/> if you want to
         /// skip mutation the children node.</remarks>
         protected virtual TBase OrchestrateChildrenMutation(TNode node, MutationContext context) =>
-            node.ReplaceNodes(node.ChildNodes(), 
-                (original, _) => MutantOrchestrator.Mutate(original, context));
+            node.ReplaceNodes(node.ChildNodes(),
+                computeReplacementNode: (original, _) => MutateSingleNode(original, context));
+
+        protected virtual SyntaxNode MutateSingleNode(SyntaxNode node, MutationContext context) => context.FindHandler(node).Mutate(node, context);
+
+        protected virtual MutationContext PrepareContext(TNode node, MutationContext context)
+        {
+            foreach (var commentTrivia in node.GetLeadingTrivia().Where(t => t.IsKind(SyntaxKind.SingleLineCommentTrivia) || t.IsKind(SyntaxKind.MultiLineCommentTrivia)).Select(t => t.ToString()))
+            {
+                // perform a quick pattern check to see if it is a 'Stryker comment'
+                if (!Pattern.Match(commentTrivia).Success)
+                {
+                    continue;
+                }
+                var match = Parser.Match(commentTrivia);
+                if (match.Success)
+                {
+                    // this is a Stryker comments, now we parse it
+                    bool mode;
```

Comment:
Mode does not feel like a bool value to me. Perhaps call it 'disable' or 'restore' or make it an Enum (we might have more modes in the future?)

Reply:
agreed, switched to `disable`